### PR TITLE
fix lint issues

### PR DESCRIPTION
**Pull request by pavelpikta**

### CODE REVIEW

**Comment by tgadiev** [comment]
Please adjust .travis.yml and .gitlab-ci.yml according to development-kit

File: README.md [hunk 1]
```diff
@@ -14,10 +17,10 @@ Requirements
 
   - Minimal Version of the ansible for installation: 2.5
 
-  -  Prepared ansible inventory file with listed VMs with python. These machines should be accessible via SSH.
+  - Prepared ansible inventory file with listed VMs with python. These machines should be accessible via SSH.
 Supported CentOS 6.* and Centos 7.*
 
-  -  oracle db client version 12.1 supported oracle server version: 12.2.0, 12.1.0, 11.2.0 (https://support.oracle.com/epmos/faces/DocumentDisplay?_afrLoop=366329397565814&id=207303.1&_afrWindowMode=0&_adf.ctrl-state=1cp26b7isr_4)
+  - Oracle db client version 12.1 supported oracle server version: 12.2.0, 12.1.0, 11.2.0
```
Comment (tgadiev) [inline README.md]: What does that mean? Sounds strange.

File: tasks/system/RedHat.yml [hunk 2]
```diff
@@ -3,12 +3,20 @@
...
   package:
     name: "{{ oracle_instantclient_artifacts }}"
     state: "present"
+  register: status
+  retries: 10
+  delay: 2
+  until: status is success
   become: True
 
 - name: "Add Oracle client to library path"
```
Comment (tgadiev) [inline tasks/system/RedHat.yml]: We have three subsequent tasks with `become: True`. How about making a block with common `become`?

**Review by tgadiev — COMMENTED** [review]
Please, see comments

File: tasks/system/RedHat.yml (hunk 2 above)
Comment (pavelpikta) [reply tasks/system/RedHat.yml]: resolved in c20dcdc

File: README.md (hunk 1 above)
Comment (pavelpikta) [reply README.md]: resolved in 8039388

**Review by tgadiev — APPROVED** [review]

**Pull request merged**